pkgs/applications/audio/reaper/default.nix

```diff
@@ -11,7 +11,12 @@
 , xdg-utils
 , which
 
-, jackSupport ? true, libjack2
+, libjack2
```

Comment:
```suggestion
```

Redundancy.

Reply:
It doesn't work without unfortunately. The `jackLibrary ? libjack2` requires a `libjack2` on its own in the pattern.
Unless you were suggesting something else that I missed?

Reply:
Oh, now I understood. You should substitute all the references of `libjack2` to `jackLibrary` and put `jackLibrary = libjack2;` in all-packages.nix.